<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rythem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background:#dddfe2;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="RyConnectionTableView" name="tableView"/>
      <widget class="QTabWidget" name="tollTabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="inspectorTab">
        <attribute name="title">
         <string>查看器</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTabWidget" name="requestInspectorTabs">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="requestInspectorTextview">
             <attribute name="title">
              <string>TextView</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QTextEdit" name="requestTextEdit"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_7">
             <attribute name="title">
              <string>webForms</string>
             </attribute>
            </widget>
           </widget>
           <widget class="QTabWidget" name="responseInspectorTabs">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="responseInspectorTextView">
             <attribute name="title">
              <string>TextView</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPlainTextEdit" name="responseTextEdit"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="responseInspectorImageView">
             <attribute name="title">
              <string>ImageView</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>523</width>
                   <height>164</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="responseInspectorHexView">
             <attribute name="title">
              <string>HexView</string>
             </attribute>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>替换规则</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWebView" name="webView">
           <property name="url">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>composer</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="Composer" name="composer" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionCapture"/>
   <addaction name="actionWaterfall"/>
   <addaction name="actionRemoveAll"/>
   <addaction name="actionLongCache"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRemoveAll">
   <property name="icon">
    <iconset resource="httpfiles.qrc">
     <normaloff>:/remove.png</normaloff>
     <normalon>:/remove.png</normalon>:/remove.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="toolTip">
    <string>清除所有</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActionCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="httpfiles.qrc">
     <normaloff>:/static/img13.png</normaloff>
     <normalon>:/static/img16.png</normalon>:/static/img13.png</iconset>
   </property>
   <property name="text">
    <string>开始抓包</string>
   </property>
   <property name="toolTip">
    <string>开始捕获</string>
   </property>
  </action>
  <action name="actionWaterfall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="httpfiles.qrc">
     <normaloff>:/static/img11.png</normaloff>:/static/img11.png</iconset>
   </property>
   <property name="text">
    <string>瀑布图</string>
   </property>
   <property name="toolTip">
    <string>瀑布图</string>
   </property>
  </action>
  <action name="actionLongCache">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="httpfiles.qrc">
     <normaloff>:/static/img10.png</normaloff>:/static/img10.png</iconset>
   </property>
   <property name="text">
    <string>长缓存</string>
   </property>
   <property name="toolTip">
    <string>被替换内容设置长时间缓存</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>Composer</class>
   <extends>QWidget</extends>
   <header>composer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RyConnectionTableView</class>
   <extends>QTableView</extends>
   <header>ryconnectiontableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="httpfiles.qrc"/>
 </resources>
 <connections/>
</ui>
